### ISSUE TITLE
hackage-security-0.5.1.0 doesn't pass Haddock stage

### ISSUE THREAD

**Issue by peti** [well-typed/hackage-security#158]
~~~~
src/Hackage/Security/TUF/Layout/Index.hs:48:5:
    parse error on input ‘-- | Package-specific metadata (@targets.json@)’
~~~~

**Comment by hvr** [comment]
It's a known issue with older Haddocks which choked on haddock-annoated GADTs; this has been fixed in Haddock 2.17

**Comment by dcoutts** [comment]
Fixed.
